texar/torch/modules/connectors/connector_base.py

```diff
@@ -63,4 +63,6 @@ def default_hparams():
 
     @property
     def output_size(self):
+        r"""The output size of :attr:`Connectors`.
```

Comment:
I don't think this one's required. Plus, there's no such thing as ``":attr:`Connectors`"``, it's either `"connectors"` or an explicit class ``":class:`SomeConnector`"``.